gabrielaEnache/src/main/java/oop/Main.java

```diff
@@ -0,0 +1,47 @@
+package oop;
+
+import oop.store.*;
```

Comment:
explicitly listing all imports

Reply:
Fixed. Had to adjust some Intellij settings for optimizing imports.